### DISCUSSION TITLE
Does the EDC support headers in an asset?

### DISCUSSION THREAD

**Discussion by anschl** [eclipse-tractusx/tractusx-edc#1188] [Q&A]
Hi everyone,

I'm trying to create an asset which needs an API key in its header. I found [some documentation](https://github.com/eclipse-tractusx/tractusx-edc/blob/040252cd7160c3148aa0bbe3282d07f762c40668/docs/usage/management-api-walkthrough/01_assets.md) but it only mentions support for `body` and `queryParams`.
I tried sending the headers in the `/bob/proxy/aas/request` request but unfortunately it returns 500 without any significant log entry in the dataplane or controlplane, so I'm not sure if it does not support it or something else went wrong.

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
